views/fe-utils/fetchUtil.js

```diff
@@ -214,6 +214,19 @@ const fetchGraphTableCSV = async (chart_id, filters, filename) => {
   return res
 }
 
+const updateUserAccess = async (user) => {
```

Comment:
updateUser it reads more of a users preferences like their avatar, username, etc.. I went with updateUserPermissions because that's what the role of the admin is doing when iteracting with the admin page.

[b780b02](https://github.com/AMP-SCZ/dpdash/pull/294/commits/b780b02b79e59c25ddadb4cd5634c5547e9dbda7)